packages/compass-crud/src/stores/crud-store.ts

```diff
@@ -2003,7 +2004,7 @@ const configureStore = (options: CrudStoreOptions & GridStoreOptions) => {
       void store.refreshDocuments();
     });
 
-    globalAppRegistry.on(
+    localAppRegistry?.on(
```

Comment:
We'd probably want to group this with other localAppRegistry events above for consistency?

Reply:
Sure, done.